### PR TITLE
Test Cases for UE Generation Handler

### PR DESCRIPTION
**Pull request by WaterMenon09**
### Introduction
This PR introduces a suite of test cases for the UE Generation Handler to ensure the accuracy and reliability of the mobility data generation process. The tests cover various aspects including the conversion of XY coordinates to geographic coordinates, the integrity of the DataFrame structure, and the consistency of data across multiple batches and ticks.

### Test Details
- *Test Conversion Accuracy*: Validates that the XY to longitude/latitude conversion aligns with expected geographical mappings.
- *DataFrame Structure Verification*: Ensures that the DataFrame contains the correct columns (mock_ue_id, longitude, latitude, tick) and that data types are appropriately assigned.
- *Batch and Tick Consistency*: Checks that data is consistent across different batches and ticks, ensuring no data leakage or misalignment occurs.

### CODE REVIEW

File: radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        self.assertEqual(boundaries, (-90.0, 90.0, -180.0, 180.0))
+
+    def test_get_gauss_markov_alpha(self):
+        alpha = UETracksGenerationHelper.get_gauss_markov_alpha(self.job_data[constants.UE_TRACKS_GENERATION][constants.PARAMS])
+        self.assertEqual(alpha, 0.8)
+
+'''
+if __name__ == '__main__':
+    unittest.main()
+'''
```
Comment (paulvarkey) [inline radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py:75]: may delete

File: radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import unittest
```
Comment (paulvarkey) [inline radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py:1]: rename file as `test_ue_tracks_generation_helper.py`

**Review by paulvarkey — CHANGES_REQUESTED** [review]
1. Feel free to add CLI command in the description that specifies how to run the tests
2. Add docstrings in each test (your PR description is quite nice -- those same comments may appear as docstrings here)

File: radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py (hunk 2 above)
Comment (WaterMenon09) [reply radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py]: renamed the file

File: radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py (hunk 1 above)
Comment (WaterMenon09) [reply radp/digital_twin/mobility/tests/test_UETracksGenerationHelper.py]: removed the redundant comments, added the docstring and added the CLI code to run in the docstring under 
"How To Run"

File: radp/digital_twin/mobility/tests/test_ue_tracks_generation_helper.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+        Mock data is used to simulate job configurations and expected outputs.
+
+        How to Run:
+        ------------
+        To run these tests, execute the following command in your terminal:
+        ```
+        python3 -m unittest radp/digital_twin/mobility/tests/test_ue_tracks_generation_helper.py
+        ```
+        """
```
Comment (paulvarkey) [inline radp/digital_twin/mobility/tests/test_ue_tracks_generation_helper.py:33]: can be moved as docstring for the class

**Review by paulvarkey — APPROVED** [review]
Approved -- please add comments from PR description as docstrings for individual tests (or, main class) as appropriate.

**Comment by WaterMenon09** [comment]
made the necessary changes and added short discrete comments for every function

**Pull request merged**